### PR TITLE
Added account_settings datasource

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Added `linode_account_settings` datasource.

### CODE REVIEW

**Comment by ezilber-akamai** [comment]
/acctest sha=d459a6a49f172c5d86b14db68f267ddafffe612c pkg=linode/accountsettings

File: website/docs/d/account_settings.html.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+The following example shows how one might use this data source to access information about Linode account settings.
+
+```hcl
+data "linode_account_settings" "example" {}
+```
+
+## Attributes Reference
+
+Each Linode LKE Version will be stored in the `versions` attribute and will export the following attributes:
```
Comment (lgarber-akamai) [inline website/docs/d/account_settings.html.markdown:23]: I think you can drop/edit this line 👍

**Review by lgarber-akamai — CHANGES_REQUESTED** [review]
Great work! Just one small change

File: website/docs/d/account_settings.html.markdown (hunk 1 above)
Comment (ezilber-akamai) [reply website/docs/d/account_settings.html.markdown:23]: Good catch! I missed that when I copy-pasted :)

**Review by lgarber-akamai — APPROVED** [review]

**Review by zliang-akamai — APPROVED** [review]

**Pull request merged**